From 0e3c4be71da7015ba00023c61fa40aea3eb1c532 Mon Sep 17 00:00:00 2001
From: Leonard Crestez <leonard.crestez@nxp.com>
Date: Wed, 9 May 2018 20:52:24 +0300
Subject: [PATCH 3767/5242] MLK-18259 arm64: dts: imx8qm: Fix mu2 interrupt in
 xen dts

commit  e2681bf35622a89338e71e6267e2a5156d4a98ce from
https://source.codeaurora.org/external/imx/linux-imx.git

This was always wrong but causes failures recently after dom0 switched
to using mu1 and claiming it's interrupt

Signed-off-by: Leonard Crestez <leonard.crestez@nxp.com>
Reviewed-by: Peng Fan <peng.fan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-mek-dom0.dts     |    2 +-
 .../boot/dts/freescale/fsl-imx8qm-mek-domu.dts     |    4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts
index 53cc032..0142147 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts
@@ -125,7 +125,7 @@
 	mu2: mu@5d1d0000 {
 		compatible = "fsl,imx8-mu";
 		reg = <0x0 0x5d1d0000 0x0 0x10000>;
-		interrupts = <GIC_SPI 177 IRQ_TYPE_LEVEL_HIGH>;
+		interrupts = <GIC_SPI 178 IRQ_TYPE_LEVEL_HIGH>;
 		fsl,scu_ap_mu_id = <0>;
 		xen,passthrough;
 		status = "disabled";
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
index 852ab8e..77908d2 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
@@ -57,10 +57,10 @@
 
 		#include "fsl-imx8qm-device.dtsi"
 
-		mu1: mu@15d1d0000 {
+		mu2: mu@15d1d0000 {
 			compatible = "fsl,imx8-mu";
 			reg = <0x1 0x5d1d0000 0x0 0x10000>;
-			interrupts = <GIC_SPI 177 IRQ_TYPE_LEVEL_HIGH>;
+			interrupts = <GIC_SPI 178 IRQ_TYPE_LEVEL_HIGH>;
 			fsl,scu_ap_mu_id = <0>;
 			status = "okay";
 		};
-- 
1.7.9.5

